resources/wafv2-ipsets.go

```diff
@@ -0,0 +1,100 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/wafv2"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+)
+
+type WAFv2IPSet struct {
+	svc       *wafv2.WAFV2
+	id        *string
+	name      *string
+	lockToken *string
+	scope     *string
+}
+
+func init() {
+	register("WAFv2IPSet", ListWAFv2IPSets)
+}
+
+func ListWAFv2IPSets(sess *session.Session) ([]Resource, error) {
+	svc := wafv2.New(sess)
+	resources := []Resource{}
+
+	params := &wafv2.ListIPSetsInput{
+		Limit: aws.Int64(50),
+		Scope: aws.String("REGIONAL"),
+	}
+
+	output, err := getIPSets(svc, params)
+	if err != nil {
+		return []Resource{}, err
+	}
+
+	resources = AppendResources(resources, output)
```

Comment:
```suggestion
	resources = append(resources, output...)
```

this should also work